docs/modelserving/v1beta1/llm/huggingface/triton_runtime.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: huggingface
+spec:
+  predictor:
+    model:
+      args:
+      - --log-verbose=1
+      modelFormat:
+        name: triton
```

Comment:
is this example for llama? triton does not support generative model, we would need to use tensorrt-llm